modules/objdetect/misc/python/test/test_qrcode_detect.py

```diff
@@ -43,10 +43,10 @@ def test_detect_and_decode_multi(self):
         retval, decoded_data, points, straight_qrcode = detector.detectAndDecodeMulti(img)
         self.assertTrue(retval)
         self.assertEqual(len(decoded_data), 6)
-        self.assertEqual(decoded_data[0], "TWO STEPS FORWARD")
-        self.assertEqual(decoded_data[1], "EXTRA")
-        self.assertEqual(decoded_data[2], "SKIP")
-        self.assertEqual(decoded_data[3], "STEP FORWARD")
-        self.assertEqual(decoded_data[4], "STEP BACK")
-        self.assertEqual(decoded_data[5], "QUESTION")
+        self.assertTrue("TWO STEPS FORWARD" in decoded_data)
+        self.assertTrue("EXTRA" in decoded_data)
+        self.assertTrue("SKIP" in decoded_data)
+        self.assertTrue("STEP FORWARD" in decoded_data)
+        self.assertTrue("STEP BACK" in decoded_data)
+        self.assertTrue("QUESTION" in decoded_data)
```

Comment:
Why? The change looks very suspicious.

Reply:
this avoids the problem of the order of the found qr codes